tests/wallet-automation/typhon/typhon-wallet-registration.spec.ts

```diff
@@ -0,0 +1,167 @@
+
+  for (let i = 0; i < seedPhrase.length; i++) {
+    const ftSeedPhraseSelector = `#app > div > div > div.flex-grow.overflow-auto > div > div.my-5.flex.justify-center.py-16 > div > div > div > div:nth-child(1) > div:nth-child(2) > div > div > div > div > div:nth-child(${i + 1}) > div > input`;
+    await newTab.waitForSelector(ftSeedPhraseSelector, { state: 'visible' });
+    await newTab.fill(ftSeedPhraseSelector, seedPhrase[i]);
+  }
+
+  const unlockWallet = '#app > div > div > div.flex-grow.overflow-auto > div > div.my-5.flex.justify-center.py-16 > div > div > div > div.mt-6.text-center.flex.justify-center > button';
+  await clickBlankSpace(newTab);
+  await newTab.waitForSelector(unlockWallet, { state: 'visible' });
+  await newTab.click(unlockWallet);
+
+  const divSelector = '//*[@id="lc"]/div[2]/div[1]/div[2]/div/div[1]/div[1]/div/div[2]/div[1]/div/span[1]';
+  await newTab.waitForSelector(divSelector, { state: 'visible' });
+
+  // use the selector to retrieve the element handle
+  const elementHandle = await newTab.$(divSelector);
+  if (elementHandle) {
+    // retrieve the text content of the element
+    const textContent = await elementHandle.textContent();
+    if (textContent !== null) {
+      // remove any formatting that might interfere with parseFloat
+      const cleanedText = textContent.replace(/,/g, '').trim();
+      const floatValue = parseFloat(cleanedText);
+      console.log('ADA:', floatValue)
+      if (!isNaN(floatValue)) {
+        if (floatValue < 500) {
+          console.log('not eligible for voting ☹️');
+        } else {
+          console.log('eligible for voting ☺');
+        }
+      } else {
+        console.log('text content is not a valid float:', textContent);
+      }
+    } else {
+      console.log('no text content found for the specified selector:', divSelector);
+    }
+  } else {
+    console.log('element not found for the specified XPath:', divSelector);
+  }
+
+  const logOut = '//*[@id="app"]/div/div/div[3]/div/div/div[1]/div/div/div[2]/div[11]/div[2]';
+  await newTab.waitForSelector(logOut, { state: 'visible' });
+  await newTab.click(logOut);
+
+  const chooseAccount = '//*[@id="app"]/div/div/div[3]/div/div[2]/div/div/div[2]/div';
+  await newTab.waitForSelector(chooseAccount, { state: 'visible' });
+  await newTab.click(chooseAccount);
+
+  const removeAccount = '//*[@id="app"]/div/div/div[3]/div/div[2]/div/div/div[2]/div[4]/button';
+  await newTab.waitForSelector(removeAccount, { state: 'visible' });
+  await newTab.click(removeAccount);
+
+  const confirmRemove = 'button.btn.bg-primary';
+  await newTab.waitForSelector(confirmRemove, { state: 'visible' });
+  await newTab.click(confirmRemove)
+
+  const addNew = '//*[@id="app"]/div/div/div[3]/div/div[2]/div/div/div[4]';
+  await newTab.waitForSelector(addNew, { state: 'visible' });
+  await newTab.click(addNew);
+  
+  // Keeping the browser open for debugging and verifying (remove the timeout or adjust as needed)
```

Comment:
Remove all commented code if you dont need it

Reply:
Removed all commented/unnecessary code for now